qep-322-profile-tool-point-edition.md

```diff
@@ -0,0 +1,60 @@
+# QGIS Enhancement 322: Add support for point layer edition from the profile tool
+
+**Date** 2025/02/13
+
+**Author** Jean Felder (@ptitjano), Jacky Volpes (@Djedouas)
+
+**Contact** jean dot felder at oslandia dot com, jacky dot volpes at oslandia dot com
+
+**Version** QGIS 3.44
+
+# Summary
+
+This proposal aims to enable the editing of point layers directly from the elevation profile. It will be possible to add new points, move existing ones, and delete them. Additionally, selecting points from the elevation profile will be possible. Editing directly within the elevation profile will make it easier to precisely adjust the elevation of 3D points.
```

Comment:
How would deletion work? Would it work on selection, or be a "click on points to delete" tool? If the latter, is it click per individual point? or marquee based?

Reply:
It will be selection based. I have added some text to explain it.